regression/CMakeLists.txt

```diff
@@ -114,6 +114,9 @@ set(REGRESSIONS_CPP11 esbmc-cpp11/cpp
                       esbmc-cpp11/template
                       )
 
+# list of C++14 test suites
+set(REGRESSIONS_CPP11 esbmc-cpp14/template)
```

Comment:
Sorry, I missed this earlier: This needs to either include "${REGRESSIONS_CPP11}" or be a new variable that's being used below.

Reply:
Very good catch, copy and paste artifact...